clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus_test.go

```diff
@@ -565,7 +573,11 @@ func TestLogging(t *testing.T) {
 			gatherer := &apiResponsivenessGatherer{}
 			config := &measurement.Config{}
 
-			gatherer.Gather(executor, time.Now(), config)
+			_, err := gatherer.Gather(executor, time.Now(), config)
+			if err != nil {
+				t.Errorf("cant gather config %v", err)
```

Comment:
This one is causing test failure. The error here is actually expected for some usecases, it shouldn't fail the test.

Reply:
So should I just mute the error over here?

Reply:
Definiely you can't fail on error.
Either muting or introducing a per test-case parameter whether an error is expected or not is the way to go.

Reply:
Removing return doesn't fix the problem - Error will still fire.

Reply:
So I will just log the error description in other format